t/lib/Bakesale/Schema/Result/User.pm

```diff
@@ -32,24 +32,48 @@ sub ix_default_properties ($self, $ctx) {
   },
 }
 
-sub ix_create_error ($self, $ctx, $error) {
+sub ix_create_error ($self, $ctx, $rec, $error) {
```

Comment:
It may be that in the future we need to pass both the record we tried to create *and* the record that the user asked us to create.  (The difference here is at least defaults.)  Maybe `$rec` should (a) some after `$error` and (b) be an entry in a hashref of arguments.

`$rclass->ix_create_error($ctx, $error, { rec => $rec })```

?